web/less/endMeeting.less

```diff
@@ -107,4 +123,86 @@ html {
       display: inline-block;
     }
   }
-}
\ No newline at end of file
+}
+
+
+.back-to-top-wrapper {
+  display:none;
+  position: absolute;
+  top: 100vh;
+    right: 3rem;
+    bottom: -22em;
+    width: 3em;
+  pointer-events: none;
+  @media @smartphonesPortrait {
+  display:block;
```

Comment:
Please indent. (But do we want this to be `display:block` in portrait mode?)

Reply:
there is some content cropped in portrait mode too. helps the user identify scrollable content. WDYT?